### PR TITLE
Add title screen buttons

### PR DESCRIPTION
**Pull request by TechStreetDev**

### CODE REVIEW

**Comment by BlazeMCworld** [comment]
you will probably want to register the title screen mixin in this pr aswell, otherwise these images dont do anything

File: src/main/resources/assets/codeutilities/lang/en_us.json [hunk 1]
```diff
@@ -172,5 +172,7 @@
   "config.codeutilities.option.dfButtonsLocation.tooltip": "The location the button text is shown at.",
 
   "config.codeutilities.option.dfButtonsLocation.top_left": "Top Left",
-  "config.codeutilities.option.dfButtonsLocation.bottom_right": "Bottom Right"
+  "config.codeutilities.option.dfButtonsLocation.bottom_right": "Bottom Right",
```
Comment (Reasonlesss) [inline src/main/resources/assets/codeutilities/lang/en_us.json:175]: do we really need this as an option?

**Review by Vatten — APPROVED** [review]

**Review by Reasonlesss — CHANGES_REQUESTED** [review]
this pr doesnt even appear to be complete?

**Comment by TechStreetDev** [comment]
> this pr doesnt even appear to be complete?

how doesnt it

**Comment by Reasonlesss** [comment]
> > this pr doesnt even appear to be complete?
> 
> how doesnt it

my bad didn't realize you made a PR earlier. I'm not sure its the best thing to do to have many different versions of each button imo

**Comment by TechStreetDev** [comment]
> > > this pr doesnt even appear to be complete?
> > 
> > 
> > how doesnt it
> 
> my bad didn't realize you made a PR earlier. I'm not sure its the best thing to do to have many different versions of each button imo

Some players wanted top left text, some wanted bottom right text some diddnt want any so /shrug i concluded with config options for them all

**Comment by Reasonlesss** [comment]
> > > > this pr doesnt even appear to be complete?
> > > 
> > > 
> > > how doesnt it
> > 
> > 
> > my bad didn't realize you made a PR earlier. I'm not sure its the best thing to do to have many different versions of each button imo
> 
> Some players wanted top left text, some wanted bottom right text some diddnt want any so /shrug i concluded with config options for them all

im not sure why its literally a number on a button?

File: src/main/resources/assets/codeutilities/lang/en_us.json (hunk 1 above)
Comment (Hammer86gn) [reply src/main/resources/assets/codeutilities/lang/en_us.json:175]: no

**Pull request merged**